src/components/BottomBar/Controls.js

```diff
@@ -131,40 +133,108 @@ export const Controls = controlsInjector(observer(({ store, history, annotation
 
     const look = (disabled || submitDisabled) ? 'disabled' : 'primary';
 
-    if ((userGenerate && !sentUserGenerate) || (store.explore && !userGenerate && store.hasInterface('submit'))) {
-      const title = submitDisabled
-        ? 'Empty annotations denied in this project'
-        : 'Save results: [ Ctrl+Enter ]';
-      // span is to display tooltip for disabled button
+    const SubmitOption = ({ isUpdate }) => {
+      return (
+        <Button
+          name="list-button"
+          look="secondary"
+        >
+          {`${isUpdate ? 'Update' : 'Submit'} and exit`}
+        </Button>
+      );
+    };
+    
+    if (isFF(FF_PROD_E_111)) {
+      const createButton = (key, title, onClickMethod, isUpdate = false) => {
+        return (
+          <ButtonTooltip key={key} title={title}>
+            <Elem name="tooltip-wrapper">
+              <Button
+                aria-label="submit"
+                name="submit"
+                disabled={disabled || submitDisabled}
+                look={look}
+                icon={(
+                  <Dropdown.Trigger
+                    content={<SubmitOption />}
+                  >
+                    <div>
+                      <LsChevron />
+                    </div>
+                  </Dropdown.Trigger>
+                )}
+                onClick={async (event) => {
+                  if (event.target.classList.contains('lsf-dropdown__trigger')) return;
```

Comment:
ideally we should use a ref here

Reply:
we are using an event not a dom selector a ref would make this over complicated

Reply:
yeah i was thinking we can if the trigger ref is the same element - but this should work fine - we can skip this